steps/cadence-genus-synthesis/scripts/main.tcl

```diff
@@ -17,20 +17,112 @@ if { $gate_clock == True } {
   set_attr lp_insert_clock_gating true
 }
  
-set_attr library    [join "
-                      [glob -nocomplain inputs/adk/*.lib]
-                      [glob -nocomplain inputs/*.lib]
-                    "]
+# ------------------------------------------------------------------------
+# (begin compiling library and lef lists)
+# ------------------------------------------------------------------------
 
-set_attr lef_library [join "
-                       inputs/adk/rtk-tech.lef
-                       [glob -nocomplain inputs/adk/*.lef]
-                       [glob -nocomplain inputs/*.lef]
-                     "]
+########################################################################
+# Library sets
+# 
+# Steveri update Aug 2020: fixed library load ordering.
+# For consistency, using code similar to what I found in
+# existing step 'cadence-innovus-flowsetup/setup.tcl'
+#
+# Also, added "lsort" to "glob" for better determinacy.
+# 
+# Note, genus at the moment only uses typical libraries;
+# I'm leaving the best/worst lists for consistency with flowsetup step,
+# plus maybe someday someone would want to design to one of these cases...?
```

Comment:
Can we remove these three lines? I think this script in particular is agnostic to which ADK view is being used.

Reply:
Done.